resources/helm/dask-gateway/templates/traefik/deployment.yaml

```diff
@@ -44,6 +44,7 @@ spec:
             - "--global.sendanonymoususage=False"
             - "--ping=true"
             - "--providers.kubernetescrd"
+            - "--providers.kubernetescrd.allowCrossNamespace=true"
             - '--providers.kubernetescrd.labelselector=gateway.dask.org/instance={{ include "dask-gateway.fullname" . }}'
             - "--providers.kubernetescrd.throttleduration=2"
```

Comment:
Hmmm, one confusion point for me is case sensitivity.

Is this accepted? Looking at https://doc.traefik.io/traefik/providers/kubernetes-crd/#allowcrossnamespace, it could make more sense if this was `--providers.kubernetesCRD.allowCrossNamespace=true` or `--providers.kubernetescrd.allowcrossnamespace=true` in a way.

I wonder if there is a silent issue with this or other flags? I saw though in the logs of traefik that this was enabled, but with a warning asking if it was intentional or not, which seems weird if we explicitly try to set it.

I'm confused about how we pass these flags overall, hmmm...

Reply:
I think everything is case insensitive, nevermind.